docs/src/examples/components/Radio/Types/RadioExample.shorthand.tsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react'
 import { Radio } from '@stardust-ui/react'
 
-const RadioExample = () => <Radio label="Make your choice" />
+const handleChange = () => {
+  console.log('The radio checked value was changed!')
```

Comment:
probably, it is better to use `alert` calls here, if the intent is to make it clear to the client that some changes have happened

Reply:
Makes sense, will change it!